resources/idlharness.js

```diff
@@ -84,10 +84,27 @@ function minOverloadLength(overloads)
     .reduce(function(m, n) { return Math.min(m, n); });
 }
 
+// A helper to get the global of a Function object.  This is needed to determine
+// which global exceptions the function throws will come from.
```

Comment:
Fun irrelevant fact, 'This is needed to determine which global exceptions the function throws will come from.' is a [garden path sentence](https://en.wikipedia.org/wiki/Garden-path_sentence). The 'global' is read as an adjective until you reach 'will come from', at which point it needs to be reparsed as a 'noun' :D.